playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/digitalgoods/DigitalGoodsConverter.java

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.androidbrowserhelper.playbilling.digitalgoods;
+
+import com.android.billingclient.api.BillingClient;
+import com.android.billingclient.api.BillingResult;
+
+import static com.google.androidbrowserhelper.playbilling.digitalgoods.Logging.logUnknownResultCode;
+
+/**
+ * This class contains common functionality that is used by the various *Call classes.
+ */
+public class DigitalGoodsConverter {
+    // These values should be in sync with the ones used here:
+    // https://source.chromium.org/chromium/chromium/src/+/master:chrome/android/java/src/org/chromium/chrome/browser/browserservices/digitalgoods/DigitalGoodsConverter.java;drc=58413e78345f4783c1e4d276300c5d283e9d3dba
+    public static final int CHROMIUM_RESULT_OK = 0;
+    public static final int CHROMIUM_RESULT_ERROR = 1;
+    public static final int CHROMIUM_RESULT_ITEM_ALREADY_OWNED = 2;
+    public static final int CHROMIUM_RESULT_ITEM_NOT_OWNED = 3;
+    public static final int CHROMIUM_RESULT_ITEM_UNAVAILABLE = 4;
+
+    private DigitalGoodsConverter() { }
+
+    static int toChromiumResponseCode(BillingResult result) {
```

Comment:
nit: this method seems to accept a more general input than necessary. Would it be cleaner to only have the version that takes an int playBillingResultCode?

Reply:
I kinda like this method because it keeps things typed - in prod (the various *Call classes), you have a responseCode and you have a billingResult. You use this method to convert from a billingResult to a responseCode. If the caller has to manually go from billingResult to billingResponseCode to responseCode, since the last two are ints there's a chance you'll just forget and not convert it.

If anything I'd get rid of the other method - but I kept that around for testing so that I don't have to bother creating a BillingResult in tests.